### ISSUE TITLE
support-v4及design依赖冲突问题

### ISSUE THREAD

**Issue by liyujiang-gzu** [yarolegovich/DiscreteScrollView#134]
需要排除discrete-scrollview的support-v4及design依赖才能正常编译，解决方案如下：
```java
    compile('com.yarolegovich:discrete-scrollview:1.4.9') {
        exclude group: "com.android.support"
    }
```

**Comment by liyujiang-gzu** [comment]
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformClassesWithJarMergingForRelease'.
> com.android.build.api.transform.TransformException: java.util.zip.ZipException: duplicate entry: android/support/design/widget/CoordinatorLayout$1.class

**Comment by yarolegovich** [comment]
Fixed with 1.5.0
